staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf/protobuf.go

```diff
@@ -71,20 +71,18 @@ func IsNotMarshalable(err error) bool {
 // as-is (any type info passed with the object will be used).
 //
 // This encoding scheme is experimental, and is subject to change at any time.
-func NewSerializer(creater runtime.ObjectCreater, typer runtime.ObjectTyper, defaultContentType string) *Serializer {
```

Comment:
@smarterclayton is there a reason for this to vary (e.g. streaming), or is this safe to hard code?

Reply:
This isn't vary in the current codebase (we have also RawSerializer for protobuf for streaming purpose) - I don't see any reason for having that.
But I'm happy to hear @smarterclayton opinion too.

Reply:
The content type used in Unknown is always content protobuf.  So this should be safe if it's going into Unknown only.